package.json

```diff
@@ -15,30 +15,25 @@
     "build-doc": "node scripts/build.js && cp build/index.html build/404.html",
     "build:copy-files": "cp -rf package.json README.md NOTICE.txt LICENSE.txt CHANGELOG.md lib",
     "build:index": "babel-node devtools/buildIndexFiles.js",
-    "build:lint:fix": "npm run build:lint -- --fix",
-    "build:lint": "eslint 'src/**' --ext .js,.jsx --env browser,node",
     "build:cjs": "cross-env NODE_ENV=production BABEL_ENV=cjs babel src --out-dir lib --ignore \"src/**/*.spec.js\",\"src/**/*.test.js\",\"src/**/*.Component.js\",\"src/_playground/*\",\"src/**/*.stories.js\"",
     "build": "npm run build:index && rm -rf lib && npm run build:cjs",
-    "config:lint": "eslint 'config/**' --ext .js,.jsx --env browser,node",
     "deploy": "gh-pages -d build",
-    "devtools:lint": "eslint 'devtools/**' --ext .js,.jsx --env browser,node",
     "docs:dev": "FUNDAMENTAL_REACT_PLAYGROUND=true npm start",
     "dry-run": "npm run build && npm run build:copy-files && npm publish lib --dry-run",
-    "lint:fix": "npm run build:lint:fix  && npm run scripts:lint:fix && npm run config:lint:fix",
+    "lint": "npm run lint:code && npm run lint:style",
+    "lint:code": "eslint . --ext .js,.jsx",
```

Comment:
Does this need the `--env browser,node` argument?

Reply:
It's already in `.eslintrc`: https://github.com/SAP/fundamental-react/blob/master/.eslintrc#L1